stories/TextLink.stories.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { withDesign } from 'storybook-addon-designs';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import { ThemeProvider } from 'styled-components';
+import { Link } from 'rebass/styled-components';
+
+import baseTheme from '../app/themes/baseTheme';
+
+const withTheme = Story => (
+  <ThemeProvider theme={baseTheme}>
+    <Story />
+  </ThemeProvider>
+);
+
+export default {
+  title: 'Text Link',
+  component: Link,
+  decorators: [withDesign, withKnobs, withTheme],
+};
+
+const linkText = () => text('Link Text', 'Link Text');
+const link = () => text('URL', '');
+
+export const TextLinkStory = () => (
+  <React.Fragment>
```

Comment:
No big deal here, but no need to wrap in `React.Fragment` since there's only one root element being rendered.

Reply:
Can change or not, at your discretion.

Reply:
makes sense to have it as simple as possible